client/assets/calendar.js

```diff
@@ -0,0 +1,121 @@
+import { saveAs } from 'file-saver'
+import Moment from 'moment'
+import {
+  extendMoment
+} from 'moment-range'
+
+export async function downloadCalendar({ plan }) {
+	const FileSaver = require('file-saver')
```

Comment:
This guy is a little outdated, either use the import statement you have above (and if that doesn't work as it may not destructure to `saveAs` properly, just do `import FileSaver from 'file-saver';`)

Reply:
Now I'm using "import FileSaver from 'file-saver';" instead of that